gef-ui/server/controllers/application-details/index.js

```diff
@@ -253,6 +253,13 @@ const applicationDetails = async (req, res, next) => {
       params.success = req.success;
     }
 
+    // Using req.flash to pass success message when facility is updated from unissued to issued. More info regarding req.flash can be found here:https://www.npmjs.com/package/connect-flash
+    const [successMessage] = req.flash('success');
+
+    if (successMessage) {
+      params.success = successMessage;
+    }
```

Comment:
Note that this should be `.message`

Reply:
If  `params.success = successMessage.message` is used  when user changes the facility from unissued to issued the functionality is broken and blank page is displayed.

Reply:
Interesting -- Ill resolve this for now